### ISSUE TITLE
About regional Dataset

### ISSUE THREAD

**Issue by Dorothylyly** [SydCaption/SAAT#20]
1. msrvtt_roi_feat.h5
2. msrvtt_roi_box.h5

this two h5 files have 10000 datasets respectively。
you just have 6513 videos ，but why you have 10000 datasets each file ？
what is the meaning of each dataset ?  is it for frames ？

**Comment by SydCaption** [comment]
Hi, the msvtt dataset actually contains 10k videos, as can be found on [http://ms-multimedia-challenge.com/2016/dataset](url). Each item in the .h5 file is the features or coordinate information of object proposals extracted from the centre frame of the input video.

**Comment by Dorothylyly** [comment]
thank you for your reply!!
do you mean that :
although msvtt contains 10k videos, you just use 6513 videos of it in your code ??

**Comment by SydCaption** [comment]
I'm sorry, but what do you mean by using 6513 videos? The official splits are 6513 in training set, 497 for validation and 2990 for test.

**Comment by Dorothylyly** [comment]
ooooooooooooooooh. 
It is my mistake, now I know it.   thank you !!!!!!!!!!!!!!!!!!!!

**Comment by Dorothylyly** [comment]
I am a novice, I forgot that there are test set and validation set
